spec/models/aws_engine_spec.rb

```diff
@@ -2,6 +2,81 @@
 require 'spec_helper'
 require 'digest/md5'
 
+module Aws
+  module SQS
+    class DummyClient
+      def send_message(params = {}, options = {})
+        nil
+      end
+      def receive_message(params = {})
+        []
+      end
+      def delete_message(params = {})
+        Seahorse::Client::Response.new
+      end
+      def delete_message_batch(params = {})
+        Aws::SQS::Types::DeleteMessageBatchResult.new
+      end
+    end
+
+    module Types
+      class DummyReceiveMessageResult
+        def initialize
+          @msg = []
+          @msg.push(DummyMessage.new)
+        end
+        def messages
+          @msg
+        end
+      end
+
+      class DummyMessage
+        attr_accessor :receipt_handle
+      end
+    end
+  end
+
+  module S3
+    class DummyClient
+      def put_object(params = {})
+
+      end
+      def get_object(params = {})
+        filename = params[:response_target]
+        File.open(filename, 'wb') { |f| f.write('hoge') }
+      end
+      def delete_objects(params = {})
+
+      end
+      def list_objects(params = {})
+        Types::DummyListObjectOutput.new
+      end
+    end
+
+    class DummyPresigner
+      def presigned_url(param1, params = {})
+        'https://signed.example.com'
+      end
+    end
+
+    module Types
+      class DummyListObjectOutput
+        def initialize
+          @contents = []
+          obj = Aws::S3::Types::Object.new
+          obj.key = 'hoge'
+          @contents.push(obj)
+        end
+
+        def contents
```

Comment:
__[rubocop]__
* Style/TrivialAccessors: Use `attr_reader` to define trivial reader methods. [:ref](https://github.com/bbatsov/ruby-style-guide#attr_family)